airflow/www/templates/airflow/ti_log.html

```diff
@@ -32,9 +32,17 @@ <h4>{{ title }}</h4>
 </ul>
 <div class="tab-content">
   {% for log in logs %}
-  <div role="tabpanel" class="tab-pane {{ 'active' if loop.last else '' }}" id="{{ loop.index }}">
-    <img id="loading-{{ loop.index }}" style="margin-top:0%; margin-left:50%; height:50px; width:50px; position: absolute;"
-         alt="spinner" src="{{ url_for('static', filename='loading.gif') }}">
+  <div
+    role="tabpanel"
```

Comment:
Is `role` a valid HTML attribute, or does this have no meaning?

Reply:
Ah, ARIA.

Reply:
Any other comments?

Reply:
Can I help with anything else?